src/huggingface_hub/inference/_client.py

```diff
@@ -1655,7 +1655,8 @@ def text_generation(  # type: ignore
         repetition_penalty: Optional[float] = None,
         return_full_text: Optional[bool] = False,  # Manual default value
         seed: Optional[int] = None,
-        stop_sequences: Optional[List[str]] = None,  # Same as `stop`
+        stop: Optional[List[str]] = None,
+        stop_sequences: Optional[List[str]] = None,  # Deprecated, use `stop` instead
```

Comment:
Note that `transformers` now uses `stop_strings` but as you mention, `transformers` will be deployed with TGI so not big deal

Reply:
Actually this will be an issue with IE deployment if not using TGI right?

Reply:
Yes and no, it's harder than that as you found out (private [gdoc](https://docs.google.com/document/d/1asPoVRbR7-H3Bd16E3A0p4sRcSZ11bQbf36BrICrLVo/edit?usp=sharing))